etna/core/fields.py

```diff
@@ -0,0 +1,236 @@
+                label=_("Year"),
+                error_messages={"incomplete": "Entered date must include a Year."},
+                validators=[
+                    PositiveIntegerStringValidator(
+                        msg="Entered date must be a real date, for example 23 9 2017."
+                    )
+                ],
+                required=default_year is None,
+            ),
+        )
+
+        super().__init__(fields=fields, **kwargs)
+
+    def clean(self, value):
+        """
+        Validate the values entered into the day, month and year fields.
+
+        Validate every value in the given list. A value is validated against
+        the corresponding Field in self.fields.
+
+        Normally, all errors are reported at the level of the MultiValueField.
+        However the Design System requires that the Error Summary has links to
+        tie an error to a specific field. To make that work the ValidationErrors
+        for each field (day, month and year) are added to a list on the respective
+        widgets as well as the error list on the (bound) field. This was the easiest
+        way to get access to the errors for a specific field when the DateInputWidget
+        on the MultiValueField is rendered.
+
+        Args:
+            value (list, tuple): the values entered into each field. The values are
+                in the order the fields are added to the ``fields`` attribute.
+
+        Raises:
+            ValidationError: if any of the values fail validation.
+
+        Returns:
+             the value converted to a ``date``.
+
+        """
+        clean_data = []
+        errors = []
+        if self.disabled and not isinstance(value, list):
+            value = self.widget.decompress(value)
+
+        if not value or isinstance(value, (list, tuple)):
+            if not value or not [v for v in value if v not in self.empty_values]:
+                # If all sub-field values are empty, don't attempt to fill in any blanks
+                if self.required:
+                    raise ValidationError(
+                        self.error_messages["required"], code="required"
+                    )
+                else:
+                    return self.compress((None, None, None))
+        else:
+            raise ValidationError(self.error_messages["invalid"], code="invalid")
+
+        for i, field in enumerate(self.fields):
+            field.widget.errors = []
+            try:
+                field_value = value[i]
+            except IndexError:
+                field_value = None
+            if field_value in self.empty_values:
+                if self.required and self.require_all_fields:
+                    # Raise a 'required' error if the MultiValueField is
+                    # required and all sub-fields are required.
+                    raise ValidationError(
+                        self.error_messages["required"], code="required"
+                    )
+
+                if (
```

Comment:
This new condition prevents a default month value from being respected below

Reply:
It does respect default month  see 
 `etna.core.tests.test_fields.DateFieldTest.test_various_inputs_with_dtf_default_for_end_date`